packages/web/src/modules/products/ProductPage.tsx

```diff
@@ -79,6 +85,30 @@ export const ProductPage = () => {
               />
             }
           >
+            {store.user?._id === data.singleProductById?.user?._id ? (
+              <Box>
+                <Button
+                  onClick={() => {
+                    handkleDeleteProduct({
```

Comment:
Just a typo here, looking good!

Reply:
Just fixed on the New PR. Thanks for the feedback!!!